Lib/test/test_perf_profiler.py

```diff
@@ -0,0 +1,348 @@
+import unittest
+import subprocess
+import sys
+import sysconfig
+import os
+import pathlib
+from test import support
+from test.support.script_helper import (
+    make_script,
+    assert_python_failure,
+    assert_python_ok,
+)
+from test.support.os_helper import temp_dir
+
+
+if not support.has_subprocess_support:
+    raise unittest.SkipTest("test module requires subprocess")
+
+
+def supports_trampoline_profiling():
+    perf_trampoline = sysconfig.get_config_var("PERF_TRAMPOLINE_SUPPORT")
```

Comment:
sysconfig provides values from `pyconfig.h`. This approach allows you to drop the Makefile variable again.

```suggestion
    perf_trampoline = sysconfig.get_config_var("_PY_HAVE_PERF_TRAMPOLINE")
```

Reply:
Nop, if we do this sysconfig doesn't include the variable.